### PR TITLE
Output update

### PR DESCRIPTION
**Pull request by laihipp**

### CODE REVIEW

**Comment by ian-mcnichols** [comment]
I'm trying to run the code and 'Display results' doesn't do anything

File: visualize.py [hunk 1]
```diff
@@ -3,36 +3,72 @@
...
+                        write.writerow(row)
+        csv_file.close()
 
 
-def build_text(output_file_name, data):
+def build_text(output_file_name, results, headers, data_type):
     if output_file_name.endswith('.txt') is False:
         output_file_name += ".txt"
     with open(output_file_name, 'w', newline='') as csv_file:
         write = csv.writer(csv_file)
```
Comment (ian-mcnichols) [inline visualize.py:47]: Is build_text writing a csv file?

File: Analyzer.py [hunk 2]
```diff
@@ -173,6 +173,7 @@ def run_probability_dist(data, datatype="Interval"):
     if datatype == "Interval":
         dist_data = data[data.dtype.names[-1]] - data[data.dtype.names[1]]
     elif datatype == "Ordinal":
+        dist_data = data[data.dtype.names[-1]] - data[data.dtype.names[1]]
```
Comment (ian-mcnichols) [inline Analyzer.py]: Remove the following lines if you're adding ordinals functionality

File: visualize.py (hunk 1 above)
Comment (laihipp) [reply visualize.py:47]: I think it's writing a comma separated .txt file. It's definitely producing a .txt fie

File: Analyzer.py [hunk 3]
```diff
@@ -169,6 +169,7 @@ def run_probability_dist(data, datatype="Interval"):
     if datatype == "Interval":
         dist_data = data[-1] - data[0]
     elif datatype == "Ordinal":
+        dist_data = data[data.dtype.names[-1]] - data[data.dtype.names[1]]
```
Comment (ian-mcnichols) [inline Analyzer.py]: This should be changed to match the data[-1] from the interval part

**Pull request closed without merging**